packages/squiggle-lang/src/rescript/Reducer/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+To interface your library there only 2 files to be modified:
```

Comment:
Please change this filename to `README.md`

Reply:
Please change the directory names from `Reducer` to `reducer`, to fit in with the other subdirectories (`camelCase` not `CamelCase`)

Reply:
The case selection was intentional. Because the filename is the module name in the source. When you make case conversions, you are able to introduce errors because of the autoconversion to module name format. I am insisting on the file naming to reflect the module naming exactly

Reply:
ok

Reply:
wait but `README.md` should be the same as it is all across github these days, though. I agree with you now about modules and directories.